site/guides/porch-ui-installation.md

```diff
@@ -22,18 +22,50 @@ This setup assumes that you have a GKE cluster up and running with porch install
 your current kube context is set to that GKE cluster. We would welcome contributions or feedback
 from people that have set this up in other clouds outside of GKE.
 
-First, find a published image in the [kpt-dev/kpt-backstage-plugins container registry](https://console.cloud.google.com/gcr/images/kpt-dev/global/kpt-backstage-plugins/backstage-plugin-cad?project=kpt-dev).
-For this example, we will use `gcr.io/kpt-dev/kpt-backstage-plugins/backstage-plugin-cad:v0.1.0`.
-
-Next, create a namespace called `backstage`:
+First, create a namespace called `backstage`:
 
 ```sh
 kubectl create namespace backstage
 ```
 
-Then, run the following command to set up the backstage deployment and service account.
-Change the image name and tag in the `newName` and `newTag` flags in the below `kpt fn eval` command to
-the ones you would like to use:
+Second, Google OAuth Credentials will need to be created to allow the UI to use
+Google authentication to authenticate users with the UI and to the Kubernetes
+API Server.
+
+To create the Google OAuth Credentials:
+
+1. Sign in to the [Google Console](https://console.cloud.google.com)
+2. Select or create a new project from the dropdown menu on the top bar
+3. Navigate to
+   [APIs & Services > Credentials](https://console.cloud.google.com/apis/credentials)
+4. Click **Create Credentials** and choose `OAuth client ID`
+5. Configure an OAuth consent screen, if required
+   - For scopes, select `openid`, `auth/userinfo.email`,
+     `auth/userinfo.profile`, and `auth/cloud-platform`.
+   - Add any users that will want access to the UI if using External user type
+6. Set **Application Type** to `Web Application` with these settings:
+   - `Name`: Config as Data UI (or any name you prefer)
+   - `Authorized JavaScript origins`: http://localhost:7007
+   - `Authorized redirect URIs`:
+     http://localhost:7007/api/auth/google/handler/frame
+7. Click Create
```

Comment:
Can a user do this step using `gcloud` CLI as well ? (not for this PR, but if it is possible, then we can share that command and save these steps).

Reply:
No :( - I looked into it yesterday. Maybe, if they are using IAP. It's not clear to me how those things interact (IAP vs OAuth 2.0 API screens above).